Header/indexHeader.css

```diff
@@ -0,0 +1,111 @@
+header {
+    /* background-image: linear-gradient(rgba(0,0,0,0.6),rgba(0,0,0,0.7)), url('https://harttohart.com/wp-content/uploads/2017/12/black-background-1468370534d5s.jpg'); */
+    height: 100vh;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    background-color: var(--bg-primary);
+}
+
+.clearfix :after {
+    content: ".";
+    visibility: hidden;
+    display: block;
+    height: 0px;
+    clear: both;
+}
+
+.main-content-header>h1 {
+    color: var(--bg-white);
+    font-family: 'Open Sans';
+}
+
+.row {
+    max-width: 1180px;
+    margin: 0 auto;
+}
+
+.logo {
+    width: auto;
+    float: left;
+    margin-top: 60px;
+    
+    
+}
+
+.main-nav {
+    float: right;
+    margin-top: 60px;
+}
+
+.main-nav li {
+    display: inline-block;
+    list-style: none;
+    margin-left: 40px;
```

Comment:
`px` is a absolute unit in CSS. It works wonders if you are creating pseudo elements, but avoid using `px` 
Reasons : 
- Your layout may break on widescreens
- It may break upon 110% + zoom
A better alternative is using `rem`

rem will use the multiplier 16px by default if not explicitly set. Rem is scalable as well.

Learn more - https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/